website/docs/md_components/button_components/usage_markdown.md

```diff
@@ -6,37 +6,37 @@ Use for basic and successful actions. They direct the user’s attention to the
 
 ### Do's
 
-- Make sure the button‘s microcopy represents the action such as “Submit“ or “Register“
-- Use an icon only when it is necessary
-- Use to perform a new action, task, move to next step
+- make sure the button‘s microcopy represents the action such as “Submit“ or “Register“
+- use an icon only when it is necessary
+- use to perform a new action, task, move to next step
 
 ### Don'ts
 
-- Don't assign an action that should be assigned for a CTA or other variations of buttons
-- Don't use an icon when it is not relevant or necessary to the action
+- don't assign an action that should be assigned for a CTA or other variations of buttons
+- don't use an icon when it is not relevant or necessary to the action
```

Comment:
keep it consistent, the 3 items above it use the full syntax do not use.
If you are explicit like in a business written format then use it everywhere
and avoid shortcuts.